lib/cocoapods/installer/xcode/pods_project_generator/pod_target_installer.rb

```diff
@@ -167,48 +167,14 @@ def add_files_to_build_phases
             end
           end
 
-          # Returns the corresponding native target to use based on the provided consumer.
```

Comment:
These were moved to `pod_target.rb` instead so I can access them from validator. No logical changes on these methods and I also moved the tests